common/history/pops/11_east_asia.txt

```diff
@@ -144,7 +144,7 @@
 		region_state:SIH = {
 			create_pop = {
 				culture = han # nanfaren
-				size = 2000000
+				size = 900000
```

Comment:
wait the population changes went wrong too?

Reply:
I changed the pop of some states by redistributing their religion/culture while keeping the total pops the same. When it merged it didn't account for that since it split off pops into Confucian as well, so I ended up with excess Han pops